modules/snooker/src/main/ts/ephox/snooker/api/Sizes.ts

```diff
@@ -19,6 +19,13 @@ const redistributeToW = function (newWidths: string[], cells: DetailExt[], unit:
   });
 };
 
+const redistributeToColumns = (newWidths: string[], groups: SugarElement[], unit: string) => {
+  Arr.each(groups, (column, index: number) => {
+    const width = Redistribution.sum([ newWidths[index] ], CellUtils.minWidth());
+    Css.set(column, 'width', width + unit);
```

Comment:
There shouldn't be too much of a risk of this, and if there is, it'd be a bug that'd affect both this mode as well as the standard mode, with the type being generated in the same place for them.
I can't speak to why we don't use Size.from. I was mainly aiming to have `redistributeToColumns` work as similarly to `redistributeToW` as was possible. Changing to Size.from here should mean we should change the `redistributeToW` as well, to avoid them operating in different and potentially disparaging ways, no?